README.md

```diff
@@ -6,14 +6,140 @@ A validator for CSV files adhering to the [Data Models](https://github.com/chop-
 
 ## Install
 
-Download the latest binary from the [releases page](https://github.com/chop-dbhi/data-models-validator/releases) for your architecture: Windows, Linux, or OS X.
+Download the latest binary from the [releases page](https://github.com/chop-dbhi/data-models-validator/releases) for your architecture: Windows, Linux, or OS X. The following examples assume the binary has been placed on your `PATH` with the name `data-models-validator`.
 
 ## Usage
 
 Run the following to see the usage.
 
 ```
-data-models-validator -help
+$ data-models-validator -help
+Data Models Validator - 0.1.0-beta.1
+
+Usage:
+
+  data-models-validator [-model <model>]
+                        [-version <version>]
+                        [-delim <delimiter>]
+                        [-compr <compression>]
+                        [-service <service>]
+                        ( <file>[:<table>]... | [:<table>] )
+
+The Data Models Validator reads a file containing data and checks it against
+the data model's schema. Input files or stream are delimited files (such as CSV)
+and optionally compressed using gzip or bzip2.
+
+One or more existing input files can be explicitly passed otherwise STDIN
+will be read. Each input can be optionally annotated with an explicit table name
+to be validated against. If not specified, the file name will be used to
+determine which table the file corresponds to.
+
+Source: https://github.com/chop-dbhi/data-models-validator
+
+Examples:
+
+  # Validate person.csv file against the OMOP v5 data model.
+  data-models-validator -model omop -version 5.0.0 person.csv
+
+  # Validate foo.csv against the person table in the OMOP v5 data model.
+  data-models-validator -model omop -version 5.0.0 foo.csv:person
+
+  # Validate the STDIN stream denoting it is tab-delimited and gzipped.
+  data-models-validator -model omop -version 5.0.0 -delim $'\t' -compr gzip
+
+```
+
```

Comment:
Hopefully the validator returns an exit status of 0 for success and nonzero otherwise, and that should be documented.